ypfs.js

```diff
@@ -0,0 +1,223 @@
+{
+	"translatorID": "5e1f3e08-ca5f-4196-a662-6cf46b3cdaa7",
+	"label": "YPFS",
```

Comment:
Can you rename the file to `YPFS.js` (rather than `ypfs.js`) to match this label?